### ISSUE TITLE
No logs written when in live mode

### ISSUE THREAD

**Issue by terryupton** [verbb/feed-me#251]
Is it possible to still generate logs when the website is Live? If I remove the 'devMode'								=>	true, from my general config file, logs are no longer written?

Is this a bug or some config I am missing?

**Comment by engram-design** [comment]
All logs should be forced whether Dev Mode is on or not, so thats quite strange. Are you running the feed via the CP, or via Cron? They should also be written when triggering via the debug action.

**Comment by idolab** [comment]
i didn't see any log in the feedme>log page too.

**Comment by johnhoysa** [comment]
I am also having issues with Logs not displaying for FeedMe. Logs seem to be working fine for Craft CMS.

**Comment by engram-design** [comment]
@idolab @johnhoysa How are you triggering your feeds, via the CP, via Cron or other means?

**Comment by terryupton** [comment]
@engram-design I am running the feeds via the CP.

**Comment by engram-design** [comment]
If you'd like to give the [latest master](https://sgroup.com.au/plugins/feedme/download) a go, I believe that'll fix this. It only seems to effect when running from the CP.

**Comment by terryupton** [comment]
@engram-design - have updated to the latest released version and I am still not seeing any logs when running via the CP.

**Comment by bstein-clever** [comment]
I'm experiencing the same problem - I've used feedme for a long time but hadn't needed the logs, so never noticed they were empty.

Today I needed to try and understand why a feed failed, but there's nothing there.

That said, I just tested, and I'm not seeing any logs in dev mode, either. Running a task with debug outut shows it processing, but nothing about logs, and nothing shows up afterwards.

Any thoughts?
